src/main/java/flight/reservation/plane/PassengerDrone.java

```diff
@@ -1,13 +1,14 @@
 package flight.reservation.plane;
 
-public class PassengerDrone {
-    private final String model;
+public class PassengerDrone extends AirCraft{
 
-    public PassengerDrone(String model) {
-        if (model.equals("HypaHype")) {
-            this.model = model;
+    public PassengerDrone(AirCraftModel airCraftModel) {
+        if (airCraftModel == AirCraftModel.HypaHype) {
+            this.model = "HypaHype";
         } else {
-            throw new IllegalArgumentException(String.format("Model type '%s' is not recognized", model));
+            throw new IllegalArgumentException(String.format("Model type '%s' is not recognized", this.model));
         }
+        this.crewCapacity = 0;
+        this.passengerCapacity = 0;
```

Comment:
same comment as in Helicopter, perhaps you can call a base class constructor here